src/utils/isObservable.js

```diff
@@ -0,0 +1,4 @@
+import $$observable from 'symbol-observable'
```

Comment:
we could have fixed is-observable as well, but yeah

Reply:
True, but that's more work than I have scope for.  (Opening the PR; writing or generating CJS, ESM, and TS versions; waiting for a merge; updating JSS when it does).  This was faster, simpler, and removes a dependency.

Reply:
On the other hand, that package could take responsibility of identifying an Observable in the future, one place to have a reliable isObservable. I am not aware if it will ever change again, but then we fix it for every one.